### PR TITLE
Address concerns about inconsistent code style and formatting

### LINKED ISSUE #27088: Code style
The example C# code should consistently place `{` on a separate line in keeping with standard C# coding conventions. Example:

```
using (reader)
{
    string? item;
    do {
        item = reader.ReadLine();
        Console.WriteLine(item);
    } while(item != null);
}
```

should be:

```
using (reader)
{
    string? item;
    do 
    {
        item = reader.ReadLine();
        Console.WriteLine(item);
    } while(item != null);
}
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 042902cc-1598-5df1-34cc-caa0fad63e02
* Version Independent ID: e186d0c4-523e-38f7-2cde-d00109ea784b
* Content: [using statement - C# Reference](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement)
* Content Source: [docs/csharp/language-reference/keywords/using-statement.md](https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/keywords/using-statement.md)
* Product: **dotnet-csharp**
* Technology: **csharp-language-reference**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

Fix code formatting / style concerns.

Fixes #27088

### CODE REVIEW

File: docs/csharp/language-reference/keywords/snippets/usings.cs [hunk 1]
```diff
@@ -27,51 +27,59 @@ private static void UsingExample()
...
             {
                 string? item;
-                do {
+                do
+                {
                     item = reader.ReadLine();
                     Console.WriteLine(item);
-                } while(item != null);
+                }
+                while (item != null);
```
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/snippets/usings.cs:35]: This is the only format I disagree with. I think putting the `while` on the next line makes it too easy to mis-read it as a stand-along `while` with a `;` as the statement. I'd consider the following instead:

```suggestion
                } while (item != null);
```

**Review by BillWagner — COMMENTED** [review]
I like this @IEvangelist 

I think one of the guidelines on the `do` ... `while` isn't what we want. What do you think?

File: docs/csharp/language-reference/keywords/snippets/usings.cs (hunk 1 above)
Comment (tdykstra) [reply docs/csharp/language-reference/keywords/snippets/usings.cs:35]: I agree, and that's how it's shown in the [language reference article](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/statements/iteration-statements#the-do-statement)

**Review by tdykstra — APPROVED** [review]
LGTM aside from the issue noted by @billwagner

File: docs/csharp/language-reference/keywords/snippets/usings.cs (hunk 1 above)
Comment (IEvangelist) [reply docs/csharp/language-reference/keywords/snippets/usings.cs]: Sounds good, will update - thanks.

**Pull request merged**